<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">


<head>
<meta name="description" content="Jinxin Xia - Research" />
<meta name="keywords" content="Ballard" />
<title> Jinxin Xia - Research </title>
<link rel="StyleSheet" href="CSS/homepage.css" title="homepage" type="text/css" />
<link rel='shortcut icon' href='favicon.ico' type='image/x-icon' />
</head>

<body>

<h1> JINXIN XIA </h1>

<hr class="navbar"/>
<div class="navbar">
<table class="navbar" align="center"><tr width="100%">
<td width="12%" align="center"><a class="navlink" href="index.html"> About </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="research.html"> Research </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="coding.html"> Coding </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="teaching.html"> Teaching </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="plots.html"> Plots </a></td></tr></table>
</div>
<hr class="navbar"/>

<div class="vspace"></div>


<div class="vspace"></div>

<div class="navbar">
<hr class="navbar"/>
</div>

<div class="vspace"></div>



<h2>Genotype Quality of Genetic Markers <br> Batch Effects for Microbiome Data</h2>

<p class="centeraligned">
<img align="middle" src="./images/AD_rank_heatmap.png" alt="(Tucker model)" width="434" height="340"/>
</p

<p class="justified">
In the first project, we detected poor genotyping of a SNP or a population trend in the regionusing run tests. 
I contributed to writing and testing codes in R in a Linux high performance computercluster environment.  
In the second project, we found a batch effects removal method that can applyto all kinds of microbiome datasets, 
whether they are balanced, skewed, tailed or not.  I proved thatcentered log transformation is not injective, 
and wrote codes in Python and R for method testing andimplementation.</p>
	

<hr class="divider">

<h2>Computation Efficiency, Algorithm </h2>

<p class="centeraligned">
<!--<img align="left" src="./images/NMF-input4000.png" alt="(original)" width=33%/>
<img align="middle" src="./images/NMF-lr4000.png" alt="(low-rank)" width=33%/>
<img align="right" src="./images/NMF-err4000.png" alt="(diff)" width=33%/>
<table width=100%><tr>
<td align="center" width=33%><b>Original</b></td>
<td align="center" width=33%><b>Low-Rank</b></td>
<td align="center" width=33%><b>Difference</b></td>
</tr></table>-->
<img align="middle" src="./images/summa.png" alt="(Hyperspectral image tree)" width="608" height="228"/>
</p>

<p class="justified">In the firstproject, I applied Gauss Newton and Levenberg-Marquardt optimization algorithms to the discrete
        Jinxin XiaBiostatistics PhD programU-M ID: 63794818optimization problem in fast matrix multiplication.  
        During this process, I also developed algorithmsfor tensor permutation and tensor derivatives. In the second project,
        I redesigned a clustering algorithmand improved the data movement efficiency in CPU by O(n<sup>0.5</sup>)</p>


<h4>Relevant Talk</h4>

<ul>
<li>
<a href="./pdfs/Fast_Matrices.pdf">Fast Matrix Multiplication Challenges and Approaches</a>.
</li>
</ul>

<hr class="divider">

<h2>Optimization Methods for Large-scale Problems in Computational Genomics</h2>


<p class="justified">I executed computation cost analysis, proved two different compact formulatedoptimization 
        methods and implemented the less computationally expensive one in MATLAB. Based onthese results, 
        I am completing a thesis about criteria of turning the Broyden Fletcher Goldfarb Shanno(BFGS) optimization 
        algorithm to the Limited-memory BFGS that could apply to all potential compactformulated BFGSs. </p>


<hr class="divider">


<div class="vspace"></div>

<hr class="navbar"/>	


</body>
</html>


