<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">


<head>
<meta name="description" content="Jinxin Xia - Research" />
<meta name="keywords" content="Ballard" />
<title> Jinxin Xia - Research </title>
<link rel="StyleSheet" href="CSS/homepage.css" title="homepage" type="text/css" />
<link rel='shortcut icon' href='favicon.ico' type='image/x-icon' />
</head>

<body>

<h1> JINXIN XIA </h1>

<hr class="navbar"/>
<div class="navbar">
<table class="navbar" align="center"><tr width="100%">
<td width="12%" align="center"><a class="navlink" href="index.html"> About </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="research.html"> Research </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="coding.html"> Coding </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="teaching.html"> Teaching </a></td>
<td width="8%">  </td>
<td width="12%"><a class="navlink" href="plots.html"> Mountains </a></td></tr></table>
</div>
<hr class="navbar"/>

<div class="vspace"></div>


<div class="vspace"></div>

<div class="navbar">
<hr class="navbar"/>
</div>

<div class="vspace"></div>



<h2>Genotype Quality Check of Genetic Markers <br> Batch Effects for Microbiome Data</h2>

<p class="centeraligned">
<img align="middle" src="./images/AD_rank_heatmap.png" alt="(Tucker model)" width="434" height="340"/>
</p>
<p class="centeraligned">
      Fig1.  Heatmap for microbiome data after batch effect removal using rank normalization
</p>


<p class="justified">

        We detected poor genotyping of a SNP or a population trend in the regionusing run tests. 
        I contributed to writing and testing codes in R in a Linux high performance computer cluster environment.  We found a batch effects removal method that can apply to all kinds of microbiome datasets, 
        whether they are balanced, skewed, tailed or not.  I proved that centered log transformation is not injective, 
        and wrote codes in Python and R for method testing and implementation.
</p>

<h4>Relevant Materials</h4>

<ul>
<li>
<a href="./pdfs/Centered log-ratio transform is not one to one.pdf">Centered log-ratio transformation is non injective proof</a>.
</li>
</ul>

<hr class="divider">

<h2>Efficient Algorithm for Clustering via Partitioned Local Depth <br> Fast Matrix Multiplication Algorithm Discovery</h2>

<p class="centeraligned">
<!--<img align="left" src="./images/NMF-input4000.png" alt="(original)" width=33%/>
<img align="middle" src="./images/NMF-lr4000.png" alt="(low-rank)" width=33%/>
<img align="right" src="./images/NMF-err4000.png" alt="(diff)" width=33%/>
<table width=100%><tr>
<td align="center" width=33%><b>Original</b></td>
<td align="center" width=33%><b>Low-Rank</b></td>
<td align="center" width=33%><b>Difference</b></td>
</tr></table>-->
<img align="middle" src="./images/summa.png" alt="(Hyperspectral image tree)" width="608" height="228"/>
</p>


<p class="centeraligned">
        Fig2.  Algorithm that minmize data movement in cache
  </p>

<p class="justified"> I applied Gauss Newton and trust optimization algorithms to discover algorithms for fast matrix multiplication.  
        During this process, I also derived efficient representation for Jacobian and
        approximate Hessian matrix and implemented them in MATLAB. In the second project,
        I redesigned a clustering algorithm and improved the data movement efficiency in CPU by O(M<sup>0.5</sup>) given the cache size of M.</p>


<h4>Relevant Materials</h4>

<ul>
<li>
<a href="./pdfs/Fast_Matrices.pdf">Fast matrix multiplication challenges and approaches</a>.
</li>
<li>
<a href="./pdfs/PaLD Algorithm.pdf">PaLD Algorithm and Improved Algorithm</a>.
</li>
<li>
<a href="./profile_results/file0.html"> Partitioned local depth algorithm MATLAB profile time analysis</a>.
</li>
<li>
<a href="./pdfs/PaLD Uxy size.pdf">PaLD Uxy size</a>.
</li>

</ul>

<hr class="divider">

<h2>Optimization Methods for Large-Scale Problems in Computational Genomics</h2>


<p class="justified">I executed computation cost analysis, proved two different compact formulated optimization 
        methods and implemented the less computationally expensive one in MATLAB. Based on these results, 
        I completed a thesis about criteria of turning the Broyden Fletcher Goldfarb Shanno(BFGS) optimization 
        algorithm to the Limited-memory BFGS that could apply to all potential compactformulated BFGSs. </p>
<hr class="divider">

<ul>
<li>
<a href="./pdfs/Presentation_Jinxin.pdf">Quasi-Newton Matrices Comparison</a>.
</li>
<li>
<a href="./pdfs/Broyden–Fletcher–Goldfarb–Shanno Optimization Algorithm Proof.pdf">Broyden–Fletcher–Goldfarb–Shanno Algorithm Proof</a>.
</li>
</ul>

<div class="vspace"></div>

<hr class="navbar"/>	


</body>
</html>


